src/main/java/org/skriptlang/skript/lang/entry/EntryValidator.java

```diff
@@ -120,23 +124,27 @@ public EntryContainer validate(SectionNode sectionNode) {
 	 * A utility builder for creating an entry validator that can be used to parse and validate a {@link SectionNode}.
 	 * @see EntryValidator#builder()
 	 */
-	public static class EntryValidatorBuilder {
+	public static class EntryValidatorBuilder extends EntryData<EntryContainer> {
```

Comment:
this isn't really appropriate design
A builder shouldn't stick around as an object itself, it should be built into the object you want
even if writing .build() is a little irritating.